core/exec.c

```diff
@@ -1070,6 +1070,30 @@ static lai_api_error_t lai_exec_reduce_op(int opcode, lai_state_t *state, struct
 
         break;
     }
+    case (EXTOP_PREFIX << 8) | STALL_OP: {
+        if (!laihost_stall && !laihost_timer)
+            lai_panic("host does not provide timer functions required by Stall()");
+
+        lai_variable_t time = {0};
```

Comment:
Shouldn't LAI variables be declared as `LAI_CLEANUP_VAR lai_variable_t name = LAI_VAR_INITIALIZER`?

Reply:
I think it is not needed for integers

Reply:
It's not needed for integers per se, but it should be done like that, to leave space to change that behaviour in the future

Reply:
Well, iirc this was copied from sleep, should I add it there as well?